roapi-http/Cargo.toml

```diff
@@ -21,8 +21,8 @@ snmalloc-rs = { version = "0.2", optional = true }
 
 # dependencies related to axum
 tokio = { version = "1", features = ["rt-multi-thread"] }
-hyper = { version = "0", features = ["http1", "server", "stream", "runtime"] }
-axum = "0.3"
+hyper = { version = "0", features = ["http1", "http2", "server", "stream", "runtime"] }
```

Comment:
I'm not :100: about this change. Adding `http2` to  the features of `axum` [results in exactly this](https://github.com/tokio-rs/axum/blob/254d8fde17555cb526f42052d23ffd4955284e98/axum/Cargo.toml#L16) so perhaps adding to `hyper`'s features is redundant?

Reply:
yes, features are additive in cargo, so if it's turned on in axum, it will be transitively turned for for hyper by axum.

Reply:
Ok I have removed the redundant addition of the feature to `hyper`